operations/locustfile.py

```diff
@@ -1,5 +1,8 @@
 from locust import HttpUser, task, events
 
+# read the sample request body for the demographics endpoint
+with open("e2e/src/test/resources/newborn_patient.json", "r") as requestBodyFile:
```

Comment:
Consider using a Locust event hook to load the data rather than just adding it at the top of the file: https://github.com/locustio/locust/blob/master/examples/test_data_management.py